packages/admin/admin-color-picker/src/PreviewIndicatorColorProps.ts

```diff
@@ -0,0 +1,7 @@
+// For reasons I cannot explain, this type must be in it's own file, otherwise the import of `ColorPickerColorPreviewProps` in the story is missing this part.
```

Comment:
Probably a cyclic dependency?

Reply:
@jamesricky I tried that locally and "fixed" it by moving the type: d9f1020fec5305678089398fa7f23ac3309751bc

Could you please verify that this works for you as well?

Reply:
Yes this fixes the issue, thanks 🙂